textpipe/doc.py

```diff
@@ -276,15 +276,15 @@ def _set_default_nlp(self, lang):
         model = spacy.load('{}_core_{}_sm'.format(lang, 'web' if lang == 'en' else 'news'))
         self._spacy_nlps[lang][None] = model
 
-    @functools.lru_cache()
+    @property
     def sentiment(self):
         """
         Returns sentiment score (-1 to 1) and a confidence level (0 to 1)
```

Comment:
Please update comments - see documentation of pattern, pattern returns a tuple of polarity and subjectivity score - this is not the same as sentiment & confidence!
https://www.clips.uantwerpen.be/pages/pattern-nl